nss-day-cohort-41/react-nutshell-the-cashews
# registration and login fully functional and styling applied

Please delete options that are not relevant.

 Bug fix (non-breaking change which fixes an issue)
 New feature (non-breaking change which adds functionality)

Testing Instructions
fetch branch to test--can now login or register and receive alert errors is info is in use or incorrect,
also styling should all appear

Checklist:
